supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -2,6 +2,18 @@
 [[code-examples]]
 = Code examples
 
+Provide the source language if it is supported by the toolchain. Do not use the Bash source language for terminal commands. Bash improperly interprets the number sign (#) as a comment instead of the prompt for a command run as the root user.
```

Comment:
Even though it's in the heading, I'd suggest adding "to code examples" for more clarity. Also more specifics on the toolchain (Red Hat Customer Portal?).

```
Provide the source language for code examples if it is supported by the Red Hat Customer Portal toolchain.
```